ke/asm.py

```diff
@@ -14,25 +14,40 @@ def maybe_wrap(self, should_wrap, regex):
         return "(?:%s)" % regex
 
 
-# python's standard re.escape()
-_special_chars_map = {
+# python's standard re.escape() with our adjustments
+_special_chars_map = {i: '\\' + chr(i) for i in b'()[]{}?*+|^$\\. \t\n\r\v\f'}
+
+_kleen_special_chars_map = {
     ord("\t"): "t",
     ord("\n"): "n",
     ord("\r"): "r",
     ord("\v"): "v",
     ord("\f"): "f",
 }
-_char_escapes = {chr(k): "\\" + v for k, v in _special_chars_map.items()}
+
+_char_escapes = {chr(k): "\\" + v for k, v in _kleen_special_chars_map.items()}
 
 
 class Literal(namedtuple("Literal", ["string"]), Asm):
     def to_regex(self, syntax, wrap=False):
         escaped = (
-            re.escape(self.string).translate(_special_chars_map).replace("\\ ", " ")
+            self.escape(self.string).translate(_kleen_special_chars_map).replace("\\ ", " ")
```

Comment:
Can we unify `pattern.translate(_special_chars_map)` and `.translate(_kleen_special_chars_map)`? They're essentially doing the same thing, right?